apps/meteor/client/components/Omnichannel/AgentOrContactCardUsername.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from '@rocket.chat/fuselage';
```

Comment:
I suggest you move this component to `apps/meteor/client/views/omnichannel/components`